application/consumer-redis-quarkus/src/main/java/webapp/tier/service/RedisSubscribeService.java

```diff
@@ -0,0 +1,75 @@
+package webapp.tier.service;
+
+import java.util.Objects;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import io.quarkus.runtime.ShutdownEvent;
+import io.quarkus.runtime.StartupEvent;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.exceptions.JedisConnectionException;
+
+@ApplicationScoped
+public class RedisSubscribeService implements Runnable {
+
+	private static final Logger LOG = Logger.getLogger(RedisSubscribeService.class.getSimpleName());
+	private final ExecutorService scheduler = Executors.newSingleThreadExecutor();
+
+	private static String servername = ConfigProvider.getConfig().getValue("redis.server", String.class);
+	private static int serverport = ConfigProvider.getConfig().getValue("redis.port.num", Integer.class);
+	private static String channel = ConfigProvider.getConfig().getValue("redis.channel", String.class);
+
+	void onStart(@Observes StartupEvent ev) {
+		scheduler.submit(this);
+		LOG.info("The application is starting...");
+	}
+
+	void onStop(@Observes ShutdownEvent ev) {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5937188)